packages/carbon-web-components/src/components/accordion/accordion.stories.ts

```diff
@@ -0,0 +1,210 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2019, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit';
+import { prefix } from '../../globals/settings';
+import { ACCORDION_SIZE } from './accordion';
+import './index';
+import './accordion-skeleton';
```

Comment:
This is included in the previous line, right?

Reply:
Yep, indeed it is. Cleaned up in https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/11306/commits/44dde0b593f8dac2c161f9a78203490182248a1b.